pyabc2/pitch.py

```diff
@@ -577,6 +599,7 @@ def from_name(cls, name: str) -> "Pitch":
         class_value = pitch_class_value(class_name)
 
         p = cls.from_class_value(class_value, octave)
+        # TODO: Why are these attributes being set? Should other from_* methods do the same?
```

Comment:
```suggestion
```
That is a great question. I realize it's a bit convoluted. It's a hack I put in to allow preserving "non-standard" pitch class name input like Cb etc., which also affects the value since the octave is set by the natural note name. For example, `Cb,` and `B,` are an octave apart.

Since you called `.from_class_name`, which calls `.from_name`, this already works.

Maybe you could add a test case or two for this.